packages/wrangler/src/type-generation/index.ts

```diff
@@ -313,6 +314,12 @@ async function generateTypes(
 		}
 	}
 
+	if (configToDTS.secret_stores) {
+		for (const secretStore of configToDTS.secret_stores) {
+			envTypeStructure.push([constructTypeKey(secretStore.binding), "?????"]);
```

Comment:
Please update this line once you have the secret store binding type defined in workers-types